src/core/server/saved_objects/migrations/integration_tests/cluster_routing_allocation_disabled.test.ts

```diff
@@ -114,7 +114,8 @@ describe('unsupported_cluster_routing_allocation', () => {
     await root.setup();
 
     await expect(root.start()).rejects.toThrowError(
-      /Unable to complete saved object migrations for the \[\.kibana.*\] index: The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue\. To proceed, please remove the cluster routing allocation settings with PUT \/_cluster\/settings {"transient": {"cluster\.routing\.allocation\.enable": null}, "persistent": {"cluster\.routing\.allocation\.enable": null}}/
+      // eslint-disable-next-line prettier/prettier
```

Comment:
For some reason, I couldn't figure out the right escapes and went with the easy option of disabling prettier here 🤷‍♀️

Reply:
I don't think you should need to disable prettier for this. Running eslint locally & it doesn't complain for me when I remove the `eslint-disable-next-line`:

```
node scripts/eslint src/core/server/saved_objects/migrations/integration_tests/cluster_routing_allocation_disabled.test.ts
```